frontend/src/styles/global.sass

```diff
@@ -0,0 +1,9 @@
+@import '~@angular/material/theming'
+@include mat-core()
+
+$app-primary: mat-palette($mat-indigo)
+$app-accent: mat-palette($mat-pink, A200, A100, A400)
+$app-warn: mat-palette($mat-red)
+$app-theme: mat-light-theme($app-primary, $app-accent, $app-warn)
```

Comment:
You can update this colors according to design)

Reply:
updated